internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -102,6 +102,10 @@ DEFAULT_DIRECTIVES_BY_PATH = {
         "gazelle:go_generate_proto false",
         "gazelle:proto_import_prefix k8s.io/apimachinery",
     ],
+    "google.golang.org/genproto/googleapis/api/annotations": [
+         "gazelle:resolve go google.golang.org/genproto/googleapis/api/annotations @org_golang_google_genproto//googleapis/api/annotations",
```

Comment:
Based on the comment you linked, it sounds like we should depend on the repo corresponding to the Go submodule, not this one. Could you confirm what's correct?

Reply:
oh, so I should replace L105 with `google.golang.org/genproto/googleapis/api` then

Reply:
or I guess we should do `google.golang.org/genproto`?